Project/src/main/resources/templates/editor.html

```diff
@@ -3,55 +3,55 @@
 <head>
     <meta charset="UTF-8">
     <title>Dashboard</title>
+    <script src="webjars/jquery/1.9.1/jquery.min.js"></script>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.css">
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.js"></script>
```

Comment:
Is there a way to import datatables as a maven dependency like we have jquery or bootstrap?

Reply:
can probably be imported from webjars, but cdn is valid way to import dependencies.